src/eko/constants.py

```diff
@@ -84,7 +87,7 @@ def uplike_flavors(nf):
     nu : int
 
     """
-    if nf not in range(2, 6 + 1):
+    if nf > 6:
```

Comment:
why do you need to change here?

Reply:
because it is used in validphys in this line https://github.com/NNPDF/nnpdf/blob/970cd427415fd6139991c2da1cb5b5ffecfa8df8/validphys2/src/validphys/photon/compute.py#L413 , in which now nf can be 0

Reply:
I see okau